<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>960</height>
   </rect>
  </property>
  <widget class="QLabel" name="camLeftLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>camLeftLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="camRightLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>camRightLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="caliPushButton">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>60</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QLabel" name="test1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="test2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>480</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takeImagesPushButton">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Take Images</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>60</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>130</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>170</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera Matrix</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="matrixPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>190</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>270</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dist Coeffis</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="coeffisPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>290</y>
     <width>251</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>370</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RMS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rmsLineEdit">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>390</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>110</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed Images</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
